### PR TITLE
Adding docker support

### PR DESCRIPTION
**Pull request by jrock2004**
Adding support for people to use `Docker` to run the server.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,13 +15,13 @@ You will need the following things properly installed on your computer.
 * [Ember CLI](http://www.ember-cli.com/)
 * [PhantomJS 2](http://phantomjs.org/)
 * [Gatekeeper](https://github.com/prose/gatekeeper)
+* [Docker](https://docs.docker.com/install/)
+* [Docker Compose](https://docs.docker.com/compose/install/)
```
Comment (Gaurav0) [inline CONTRIBUTING.md]: These should not be in the general list of prerequisites, as it is not necessary to use docker.

**Comment by jrock2004** [comment]
Want to add optional text in there?

File: CONTRIBUTING.md (hunk 1 above)
Comment (jrock2004) [reply CONTRIBUTING.md]: Sure I can figure something out

**Comment by jrock2004** [comment]
Not sure why the tests would be failing as I changed no code. All I changed was the contribute file

**Pull request merged**